src/main/java/com/google/api/codegen/util/ruby/RubyCommentReformatter.java

```diff
@@ -17,17 +17,44 @@
 import com.google.api.codegen.CommentPatterns;
 import com.google.api.codegen.util.CommentReformatter;
 import com.google.common.base.Splitter;
+import com.google.common.base.Strings;
 import java.util.regex.Matcher;
 
 public class RubyCommentReformatter implements CommentReformatter {
   @Override
   public String reformat(String comment) {
```

Comment:
It handles the whole reformating, including all the other `reformatXXX` methods in `applyTransformations`.